coverage_config_x86_64.json

```diff
@@ -1,5 +1,5 @@
 {
-  "coverage_score": 91.6,
+  "coverage_score": 91.4,
```

Comment:
I think this is dropping because of the removed TotalSizeTooLarge test case; it is not really practical to trigger this error now without allocating a large amount of memory (4GB+), so I'm not sure what the best option would be.

Reply:
I ran kcov locally and the only thing uncovered are the errors, so I think it's fine.